k8s/prod/common/divorce/div-cos.yaml

```diff
@@ -31,6 +31,7 @@ spec:
       environment:
         MANAGEMENT_ENDPOINT_HEALTH_CACHE_TIMETOLIVE: "4000"
         IDAM_API_URL: "https://idam-api.platform.hmcts.net"
+        SPRING_MAIL_PROPERTIES_MAIL_SMTP_STARTTLS_ENABLE: false
```

Comment:
This was true though https://github.com/hmcts/div-case-orchestration-service/blob/master/infrastructure/variables.tf#L139 (I cannot see it overridden in prod in *.tf)

Reply:
That's true. The thing is that it's the only difference I can see between AAT (which works) and PROD (which doesn't)

Reply:
Okay worth a try I guess..